src/EthernetDriver.cpp

```diff
@@ -0,0 +1,32 @@
+#include "EthernetDriver.h"
+
+namespace ethernet_driver {
+
+EthernetDriver::EthernetDriver() : clientIP(CLIENT_IP_ADDR) {
+}
+
+void EthernetDriver::initDriver() {
```

Comment:
Why isn't this just part of the constructor? Is there some reason that initialization might be worth waiting on in certain situations after the object is made?

Reply:
I want to keep initialization of software objects and hardware separate. Hardware often has to be initialized in a particular order. Separating the hardware and software initialization avoids issues of inadvertently calling the constructor and trying to initialize the hardware before other hardware is properly set up (i.e. designating pins as input or output, for instance).